L1Trigger/L1TTrackMatch/interface/DisplacedVertexProducer.h

```diff
@@ -0,0 +1,179 @@
+#ifndef __L1Trigger_L1TTrackMatch_DisplacedVertexProducer_h__
+#define __L1Trigger_L1TTrackMatch_DisplacedVertexProducer_h__
+
+#include "DataFormats/L1Trigger/interface/DisplacedVertex.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTrack.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTrack_TrackWord.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "SimDataFormats/Associations/interface/TTTrackAssociationMap.h"
+#include "TMath.h"
+#include <iostream>
+#include <map>
+#include <set>
+#include <string>
+#include <vector>
+#include <valarray>
+#include <ap_int.h>
+#include "conifer.h"
+
+using namespace std;
+
+class Track_Parameters {
+public:
+  float pt;
+  float d0;
+  float z0;
+  float eta;
+  float phi;
+  float charge;
+  float rho;
+  int index;
+  float x0;
+  float y0;
+  int nstubs;
+  float chi2rphi;
+  float chi2rz;
+  float bendchi2;
+  float MVA1;
+
+  float z(float x, float y) {
```

Comment:
This function should have a more descriptive name and not just a single letter

Reply:
Changed the name to TrackZAtVertex in https://github.com/cms-sw/cmssw/commit/276ecb95fae3fd47e9d4eeb7ab7c722bd846d158